### DISCUSSION TITLE
Graphql endpoint request complaint about "was submitted too quickly"

### DISCUSSION THREAD

**Discussion by rlnchow** [community/community#36708] [Apps, API and Webhooks] [answered]
I'm trying to compare different repositories and create repositories using the python gql library. A few requests returned the error "was submitted too quickly."
Is there a way I can allow multiple requests? I know I didn't reach the limit. Is there a limit per min or hour?

**Comment by LangLangBart** [comment, marked as answer]
Hi @rlnchow,
probably a question for an actual GitHub employee, but I encountered the same error as well and the answers given by `mislav` (is an actual GitHub employee) in the linked thread below shed some light on the problem.
<img src="https://user-images.githubusercontent.com/92653266/196818788-f4ad614d-3d78-4df3-8dd1-2a5725afd082.png" width="800">

[ #4801 cli/cli - create issue fails with "GraphQL error: was submitted too quickly"](https://github.com/cli/cli/issues/4801)

> mislav: Okay so I've spoke to people internally and it looks like that, in addition to general API rate limit for queries (which you are not hitting), there is content creation rate limit for resources like Issues.

PS: Maybe caching can help a little in comparing different repositories ([GitHub Docs - rate-limit](https://docs.github.com/en/rest/overview/resources-in-the-rest-api#staying-within-the-rate-limit)). `¯\_(ツ)_/¯`
